CMakeLists.txt

```diff
@@ -98,8 +101,8 @@ endif ()
 #
 # Check compiler version
 #
-set_mininum_cxx_compiler_version(GNU 4.8)
-set_mininum_cxx_compiler_version(MSVC 19.23)
+set_mininum_compiler_version(CXX GNU 4.8)
```

Comment:
probably higher now since we require C++14 in AMReX :)
@WeiqunZhang what's the oldest GCC we currently support?

Reply:
The oldest I have been using is gcc 5.4.  But the officially it looks like gcc 6.1 is the first one that fully supports C++14. https://gcc.gnu.org/onlinedocs/libstdc++/manual/status.html#status.iso.2014

Reply:
Should I go with 6.1 then?

Reply:
I think we should go with 5.1.  https://www.gnu.org/software/gcc/projects/cxx-status.html#cxx14

Reply:
Done